src/Microsoft.OData.Edm/Vocabularies/Annotations/EdmVocabularyAnnotation.cs

```diff
@@ -44,8 +49,215 @@ public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term,
             this.term = term;
             this.qualifier = qualifier;
             this.value = value;
+            this.usesDefault = false;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the EdmVocabularyAnnotation class
+        /// to the default value.
+        /// </summary>
+        /// <param name=“target”>Element the annotation applies to.</param>
+        /// <param name=“term”>Term bound by the annotation</param>
+        public EdmVocabularyAnnotation(IEdmVocabularyAnnotatable target, IEdmTerm term)
+        {
+            // Check arguments
+            if (term.DefaultValue == null) // throw error if no default value
+            {
+                // Note: why is this string not okay but the one in LiteralFormatter.cs is?
+                // Should I create the following in Microsoft.OData.Core.cs?
+                // internal const string EdmVocabularyAnnotations_DidNotFindDefaultValue = "EdmVocabularyAnnotations_DidNotFindDefaultValue";
+                //throw new ODataException("Type name should not be null or empty when serializing an Enum value for URI key.");
+                EdmUtil.CheckArgumentNull(new EdmStringConstant(null), "value");
+            }
+            EdmUtil.CheckArgumentNull(target, "target");
+            EdmUtil.CheckArgumentNull(term, "term");

```

Comment:
>EdmUtil.CheckArgumentNull(term, "term"); [](http://example.com/codeflow?start=12&length=40)

EdmUtil.CheckArgumentNull(term, "term"); should go before 

if (term.DefaultValue == null)
{...}

Reply:
Done ✅